### PR TITLE
add github actions  for  code coverage, switch var

### PR DESCRIPTION
**Pull request by lfbdev**
# Summary

add github actions  for moving issues in Trello
2 files changed

### Related Issues

#13  Add More Support for Golang for GitHub Actions Plugin - Implementation

### CODE REVIEW

**Comment by IronCore864** [comment]
- add a custom build command for golang (`make build`)
- add a switch for coverage

**Comment by IronCore864** [comment]
We can move Trello to a new plugin. We might need to support Jira as well.

File: internal/pkg/plugin/githubactions/golang/templates.go [hunk 1]
```diff
@@ -1,11 +1,21 @@
 package golang
 
-var PrBuilder = `
-name: PR Builder
+var PrMasterBuilder = `
```
Comment (IronCore864) [inline internal/pkg/plugin/githubactions/golang/templates.go:3]: Change variable name and job name to a better one.

File: internal/pkg/plugin/githubactions/workflow.go [hunk 2]
```diff
@@ -12,6 +11,7 @@ const (
 	defaultCommitMessage = "builder by DevStream"
 	BuilderYmlPr         = "pr-builder.yml"
 	BuilderYmlMaster     = "master-builder.yml"
+	BuilderYmlPrMaster   = "pr-master-builder.yml"
```
Comment (IronCore864) [inline internal/pkg/plugin/githubactions/workflow.go:14]: Maybe we need a better name, with "-devstream.yaml" as suffix.

Comment (IronCore864) [reply internal/pkg/plugin/githubactions/workflow.go:14]: Or maybe we don't need the suffix; only a better name.

File: internal/pkg/plugin/githubactions/golang/templates.go (hunk 1 above)
Comment (lfbdev) [reply internal/pkg/plugin/githubactions/golang/templates.go]: done, update it to "PipelibeBuilder"

File: internal/pkg/plugin/githubactions/workflow.go (hunk 2 above)
Comment (lfbdev) [reply internal/pkg/plugin/githubactions/workflow.go]: done

**Review by IronCore864 — APPROVED** [review]

**Review by daniel-hutao — APPROVED** [review]
/lgtm
/approve

**Comment by IronCore864** [comment]
Closes #13

**Pull request merged**